schema.graphql

```diff
@@ -302,6 +316,20 @@ input ItineraryOneWayInput {
   outboundDate: DateRange!
 }
 
+input ItineraryReturnInput {
+  """Origin of the flight"""
```

Comment:
Pls, Can you avoid using the flight in the description, because in the future margarita will support also buses and trains?

Reply:
Done!